tox.ini

```diff
@@ -1,8 +1,5 @@
 [tox]
-envlist = 
-    py27
-    py35
-    py36
+envlist = ALL
```

Comment:
I'd keep this as an enumerated list of the Python versions that we declare support for. That way folks using `tox` locally know which python interpreters they need installed on their machines.

Reply:
Ah, good point.